packages/x-components/src/x-modules/tagging/service/external-tagging.service.ts

```diff
@@ -3,25 +3,32 @@ import { Result } from '@empathyco/x-types';
 import { BrowserStorageService, StorageService } from '@empathyco/x-storage-service';
 import { RootXStoreState } from '../../../store/index';
 import { XPlugin } from '../../../plugins/index';
-import { PDPAddToCartService } from './types';
+import { ExternalTaggingService } from './types';
 
 /**
- * Default implementation for the {@link PDPAddToCartService}.
+ * Default implementation for the {@link ExternalTaggingService}.
  *
  * @public
  */
-export class DefaultPDPAddToCartService implements PDPAddToCartService {
+export class DefaultExternalTaggingService implements ExternalTaggingService {
   /**
-   * Session id key to use as key in the storage.
+   * Session id key to use as key in the storage for result clicks.
```

Comment:
I would add here the PDP context to make clear we are browsing there:
```suggestion
   * Session id key to use as key in the storage for result clicks to browse to PDP.
```

Reply:
Not sure if it helps to add that because it could lead to confusion. Maybe a click on the result doesn't browse to PDP. We are just saving the info on result clicks. What we do with that info is out of this very concrete scope. WDYT?

Reply:
OK, it makes sense to not to have it coupled to the browsing scenario